store/index.js

```diff
@@ -253,10 +260,19 @@ export const actions = {
         }))]);
     },
 
-    async [$actions.downloadAvatar]({ commit, state }, { jid }) {
+    /**
```

Comment:
Good stuff documenting everything. I think we should arrange the actual store variables/getters/setters better and maybe add some comments to categorize them better, it's a bit of a mess as is.